src/app/record/components/funding-stacks-groups/modals/modal-funding-search-link/modal-funding-search-link.component.html

```diff
@@ -22,7 +22,9 @@
     </div>
   </div>
   <div class="row">
-    <h3 i18n="@@shared.availableSearch">Available Search & Link wizards</h3>
+    <h3 i18n="@@shared.availableSearch">
+      Available Search & Link wizards ({{ total }})
```

Comment:
You will need to move the i18n tag into a `ng-container` So it does not include the `({{total}})` part 
```suggestion
      <ng-container i18n="@@shared.availableSearch">Available Search & Link wizards</ng-container> ({{ total }})
```